File: Modul-5/Soal 5/PRAK505-2410817120019-NadyaRamadhani.c
#include <stdio.h>

void Biodata(int tahunLahir, char Namaku[20], char Asal[15]){
    int tahun_sekarang = 2020;
    int usia = tahun_sekarang - tahunLahir;
    printf("Perkenalkan Nama Saya : %s\n", Namaku);
    printf("Umur Saya : %d\n", usia);
    printf("Saya Adalah Angkatan : 2020\n");
    printf("Asal Saya dari : %s\n", Asal);
}

int main(){
    int tahunLahir;
        char Namaku[20], Asal[15];
        scanf(" %d", &tahunLahir);
        scanf(" %[^\n]%*c", &Namaku);
        scanf(" %[^\n]%*c", &Asal);
    Biodata(tahunLahir, Namaku, Asal);
    return 0;
}

File: Modul-5/Soal 5/PRAK505-2410817120019-NadyaRamadhani.py
def Biodata(tahunLahir, Namaku, Asal):
    tahun_sekarang = 2020
    usia = tahun_sekarang - tahunLahir
    print(f"Perkenalkan Nama Saya : {Namaku}")
    print(f"Umur Saya : {usia}")
    print("Saya Adalah Angkatan : 2020")
    print(f"Asal Saya dari : {Asal}")

tahunLahir = int(input())
Namaku = input()
Asal = input()
Biodata(tahunLahir, Namaku, Asal)